main.tf

```
@@ -5,7 +5,7 @@ terraform {
       version = ">= 3.24.1"
     }
   }
-  required_version = "~> 0.14"
+  required_version = "~> 0.15"
```

```suggestion
  required_version = ">= 0.15"
```

So this tutorial works if you're using v1.0 or later